src/Core/View.js

```diff
@@ -364,48 +362,96 @@ View.prototype.getLayers = function getLayers(filter) {
 };
 
 /**
- * @typedef {object} FrameRequester
- * @property {Function(dt, updateLoopRestarted)} update - Method that will be called each
- * time the MainLoop updates. This function will be given as parameter the
- * delta (in ms) between this update and the previous one, and whether or not
- * we just started to render again. This update is considered as the "next"
- * update if view.notifyChange was called during a precedent update. If
- * view.notifyChange has been called by something else (other micro/macrotask,
- * UI events etc...), then this update is considered as being the "first".
+ * @name FrameRequester
+ * @function
+ * Method that will be called each time the MainLoop updates (FrameRequester can
+ * also be a function). This function will be given as parameter the delta (in
+ * ms) between this update and the previous one, and whether or not we just
+ * started to render again. This update is considered as the "next" update if
+ * view.notifyChange was called during a precedent update. If view.notifyChange
+ * has been called by something else (other micro/macrotask, UI events etc...),
+ * then this update is considered as being the "first". It can also receive
+ * optional arguments, depending on the attach point of this function.
+ * Currently only BEFORE_LAYER_UPDATE / AFTER_LAYER_UPDATE attach points provide
+ * an additional argument: the layer being updated.
  *
- * This means that if a FrameRequester.update function wants to animate
- * something, it should keep on calling view.notifyChange until its task is
- * done.
+ * This means that if a frameRequester function wants to animate something, it
+ * should keep on calling view.notifyChange until its task is done.
  *
- * Implementors of FrameRequester.update should keep in mind that this function
- * will be potentially called at each frame, thus care should be given about
+ * Implementors of frameRequester should keep in mind that this function will be
+ * potentially called at each frame, thus care should be given about
  * performance.
  *
- * Typical FrameRequesters are controls, module wanting to animate moves or UI
+ * Typical frameRequesters are controls, module wanting to animate moves or UI
  * elements etc... Basically anything that would want to call
  * requestAnimationFrame.
+ *
+ * @param {number} dt
+ * @param {boolean} updateLoopRestarted
+ * @param {...*} args
  */
 /**
  * Add a frame requester to this view.
  *
  * FrameRequesters can activate the MainLoop update by calling view.notifyChange.
  *
- * @param {FrameRequester} frameRequester
- * @param {Function} frameRequester.update - update will be called at each
- * MainLoop update with the time delta between last update, or 0 if the
- * MainLoop has just been relaunched.
+ * @param {String} when - decide when the frameRequester should be called during
```

Comment:
Could you document the possible values?  You can list them here, or add jsdoc when the constants are defined and reference it here, as you like

Reply:
Just updated it !

Reply:
@autra it's ok for you?